### DISCUSSION TITLE
More Characters for Sonic Worlds Next

### DISCUSSION THREAD

**Discussion by UltramanTime** [Techokami/SonicWorldsNext#49] [Ideas]
Hello Buddy
Here's my idea for Sonic Worlds Next

More Characters for Sonic Worlds Next (w/ Abilities):

Silver the Hedgehog:
Levitate
Psychokinesis

Mighty the Armadillo:
Hammer Drop
Wall Jump

Ray the Flying Squirrel:
Air Glide

Vector the Crocodile:
Air Dash
Wall Climb (While Air Dash)

Espio the Chameleon:
Double Jump
Shuriken

Charmy Bee:
Dash

Metal Sonic:
Black Shield
Super Peel-Out
V. Maximum Overdrive Attack

Blaze the Cat:
Axel Tornado
Burst Hover
Fire Boost

Rouge the Bat:
Glide/Climb
Bat Bomb

Cream the Rabbit:
Flying
Chao Attack

Big the Cat:
Lure Attack
Body Slam
Float
Upward Dash

Sally Acorn:
NICOLE Laser Attack
NICOLE Scan Mode
NICOLE Downward Laser
Slide 
Somersault Dash
Upward Dash

Bunnie Rabbot:
Punch
Air Punch (In Air)
Jet Hover
Rev Dash

Somari:
Fireball
Ground Pound

Honey the Cat:
Flutter Jump
Meow Sprint

Sonia the Hedgehog:
Super Twirl

Manic the Hedgehog:
Drumstick Throw

**Comment by DimensionWarped** [comment]
I would suggest sticking to one fleshed out proposal at a time and if you can, include a pull request with it... while also being preparred for the very high likelihood of it being rejected. Keep in mind that adding a new character is a lot of work in no small part because it requires the new character to get 250-300 frames of animation made. Last time I commissioned some character animation work for this project, I think I spent around $15 / frame... so that should give you a brief estimate of the value of the work needed to make a new character.

Throwing out a couple of move names is not especially useful to actually getting a character made. Also most of these characters don't really fit with the goals of the project. And as far as I'm aware there are no existing sprite sheets of any of them that are in the style of Sonic 2/3 and have a good portion of the frames needed other than maybe Mighty and Ray... who would require significant effort to change the visual style from Mania to something closer to Sonic 2/3.

If you want to get involved in the effort, doing that sprite conversion be a great place to start. Might and Ray are the best candidates for inclusion in that list followed at a sizable distance by the Chaotix (as they existed in Knuckles Chaotix specifically, but those sprites are definitely lacking in necessary animation).

**Comment by Techokami** [comment]
Pretty much what DW said. While I do have some long term plans for three to four potential characters to add, they will mainly be in the form of tutorials to guide people in the process of creating their own characters for their projects, rather than something pre-fab.

**Comment by naoshibaikyu** [comment]
This is asking for *a lot* of characters. I don't think they're opposed to more characters but this is beyond a reasonable scope. And not only that, but threads like this isn't very helpful because you're just listing characters and their abilities without explaining what each of them do.
